project/src/main/java/com/google/sps/data/Place.java

```diff
@@ -143,6 +149,12 @@ public static Builder builder() {
      */
     public abstract Builder setBusinessStatus(BusinessStatus businessStatus);
 
+    /**
+     * @param cuisines the place's cuisines.
+     * @return a Place builder that enables to continue building
+     */
+    public abstract Builder setCuisines(ImmutableList<String> cuisines);
```

Comment:
I started to change it but then I realized that the tests are already built with a cuisines immutable list and that it is needed this way for other uses, so this change will require to change more classes. I will keep it mind for next times because I see how it simplifies the tests.